README.md

```diff
@@ -372,6 +372,29 @@ Opposite of `isBlank`
 Recurses through an object's leaf properties and passes an array of booleans to the combinator, such as `_.some`, `_.every`, and `F.none`
 
 
+## Promises
+
+## asyncMap
+Like lodash/fp map, but accepts promises. Promises run unordered.
+
+## asyncSeries
+Like lodash/fp map, but accepts promises. Promises run in order.
+
+## asyncReduce
+Like lodash/fp reduce, but accepts promises. Promises run in order.
+
+## asyncProps
```

Comment:
Async Props isn't int here, but it's also quite literally Promise.props from bluebird